### PR TITLE
Add Sticky Notes

### PR DESCRIPTION
**Pull request by vixalien**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.vixalien.sticky.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        {
+            "name" : "yarn-deps",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "/usr/lib/sdk/node18/enable.sh",
+                "mkdir -p /app",
+                "cp -r $FLATPAK_BUILDER_BUILDDIR/flatpak-node/yarn-mirror/ /app"
+            ],
+            "sources" : [
+                "generated-sources.json"
```
Comment (hfiguiere) [inline com.vixalien.sticky.json:45]: this file is missing

File: com.vixalien.sticky.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        {
+            "name" : "sticky-notes",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dyarnrc=/app/.yarnrc"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/vixalien/sticky/archive/refs/tags/0.1.0.zip"
```
Comment (hfiguiere) [inline com.vixalien.sticky.json:64]: a checksum is required.

**Comment by vixalien** [comment]
ptal

File: com.vixalien.sticky.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    "app-id" : "com.vixalien.sticky",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.node18"
+    ],
+    "tags" : [
+        "nightly"
+    ],
```
Comment (hfiguiere) [inline com.vixalien.sticky.json:11]:
```suggestion
```

**Comment by hfiguiere** [comment]
bot, build com.vixalien.sticky

**Comment by vixalien** [comment]
ptal

**Pull request closed without merging**